lib/cereal/serializer.ex

```diff
@@ -60,6 +58,14 @@ defmodule Cereal.Serializer do
     end
   end
 
+  defp define_default_transform do
+    quote do
+      def transform(data), do: data
```

Comment:
Do we even need the 1-arity function anymore?

Reply:
we do use it in one other place in massdriver so I figured we should make it backwards compatible.. although I could also update that function call

Reply:
I think it would be slightly easier to have just the 2-arity function. This is also inline with most of the other overrideables.

Reply:
updated